src/applications/vaos/reducers/appointments.js

```diff
@@ -124,7 +124,8 @@ export default function appointmentsReducer(state = initialState, action) {
           (acc, clinic) => ({
             ...acc,
             [`${clinic.systemId}_${clinic.locationIen}`]: facilityData[
-              getRealFacilityId(clinic.institutionCode)
+              // This var prefix should be removed when appointment list is updated to FHIR
```

Comment:
FYI @narin

Reply:
Let's coordinate on this depending on which of our PRs gets merged first.

Reply:
@narin I think it'll be easier if your PR is merged first. That should give me the opportunity to see if having both this and the appointment list converted will allow us to remove some of the parse calls.